megaphone-validator/lib/reupload-audio.js

```diff
@@ -21,9 +21,9 @@ module.exports = (megaphoneEpisode, audioUrl) => {
             url: `https://cms.megaphone.fm/api/networks/${networkId}/podcasts/${podcastId}/episodes/${episodeId}`,
         };
 
-        request(options, (err, response, body) => {
-            if (err) reject(err);
-            resolve(body);
+        request(options, (error, response, body) => {
+            if (error) reject({ level: "ERROR", message: error });
+            resolve({ level: "REUPLOADED", message: `${programSlug}'s audio file was reuploaded`, body });
```

Comment:
Typically, the levels are like `INFO` `WARNING` or `ERROR`.  Maybe this one could be `WARNING` since it failed gracefully but might still be more "errory" than `INFO`?

Reply:
@nakedsushi 
Ah! Yes that makes complete sense. I'll go ahead and adjust this to `WARNING`, and then that way if we ever need warning for something else we can use that bucket for that too.